uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/apis/leave-request-api-mock.js

```diff
@@ -31,12 +31,19 @@ define([
         },
         calculateBalanceChange: function (params) {
           return $q(function (resolve, reject) {
+            if(params && params.from_type == 'half_day_am'){
```

Comment:
`params` should not be checked twice,
string equality should be checked using `===`

```
if(params) {
  if(params.from_type === 'half_day_am') {
    resolve(mockData.singleDayCalculateBalanceChange().values);
  }
  else if(params.from_type === 'all_day') {
    resolve(mockData.multipleDayCalculateBalanceChange().values);
  }
}
```